### PR TITLE
Allow packages from "symfony/*" ^7

### PR DESCRIPTION
**Pull request by phansys**
## Subject

I am targeting this branch, because this change respects BC.

## Changelog

```markdown
### Added
- Support for packages from `symfony/*` 7.x
```

## To do

- [x] Wait for the resolution in shivammathur/setup-php#799.

~~I don't know why we need to disable the "psr" extension for the "composer" CI job in this repository, so I left this change in a separated commit.
Please, let me know if you have any concern or insight about this.~~

### CODE REVIEW

File: .github/workflows/lint.yaml [hunk 1]
```diff
@@ -54,6 +54,7 @@ jobs:
               uses: shivammathur/setup-php@v2
               with:
                   php-version: '8.2'
+                  extensions: ':psr'
```
Comment (VincentLanglet) [inline .github/workflows/lint.yaml]: If we need to do this, this require a dev-kit PR

Comment (phansys) [reply .github/workflows/lint.yaml]: Yes, I know, but I'm not sure why do we need this change.
I'll create the PR on dev-kit later :+1:

Comment (phansys) [reply .github/workflows/lint.yaml]: See shivammathur/setup-php#799.
I guess we should wait to confirm if the extension is disabled in that project or implement the solution on our side.

**Review by dmaicher — APPROVED** [review]

**Review by VincentLanglet — APPROVED** [review]

**Comment by VincentLanglet** [comment]
Thanks

**Pull request merged**